nemo/collections/nlp/modules/common/megatron/transformer.py

```diff
@@ -58,6 +60,24 @@
 """
 
 
+class ColumnLinear(tensor_parallel.ColumnParallelLinear):
```

Comment:
Why is this needed?

Reply:
tensor_parallel.ColumnParallelLinear won't export - it has Autograd Function in execution path during inference - which is wrong and I have reported the issue to Apex - but we need some workaround in place now.